cmd/podman/networks/list.go

```diff
@@ -78,13 +79,37 @@ func networkList(cmd *cobra.Command, args []string) error {
 	}
 
 	switch {
-	case report.IsJSON(networkListOptions.Format):
-		return jsonOut(responses)
+	// quiet means we only print the network names
 	case networkListOptions.Quiet:
-		// quiet means we only print the network names
 		return quietOut(responses)
+
+	// TODO remove references to parse package
+	case parse.MatchesJSONFormat(networkListOptions.Format):
+		return jsonOut(responses)
+
+	// table or other format output
+	default:
+		return templateOut(responses, cmd)
+	}
+}
+
+func quietOut(responses []*entities.NetworkListReport) error {
+	for _, r := range responses {
+		fmt.Println(r.Name)
+	}
+	return nil
+}
+
+func jsonOut(responses []*entities.NetworkListReport) error {
+	prettyJSON, err := json.MarshalIndent(responses, "", "  ")
```

Comment:
You should not import encoding/json. Add the following to network.go and just use the package global.

```
	// Pull in configured json library
	json = registry.JSONLibrary()
```